apps/test/unit/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditorTest.jsx

```diff
@@ -3,10 +3,32 @@ import {mount} from 'enzyme';
 import sinon from 'sinon';
 import {expect} from '../../../../util/reconfiguredChai';
 import {UnconnectedProgrammingExpressionsEditor as ProgrammingExpressionsEditor} from '@cdo/apps/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditor';
+import {Provider} from 'react-redux';
+import {
+  stubRedux,
+  restoreRedux,
+  getStore,
+  registerReducers
+} from '@cdo/apps/redux';
+import reducers, {
+  init
+} from '@cdo/apps/lib/levelbuilder/lesson-editor/activitiesEditorRedux';
 
 describe('ProgrammingExpressionsEditor', () => {
-  let defaultProps, removeProgrammingExpression;
+  let defaultProps,
+    addProgrammingExpression,
+    removeProgrammingExpression,
+    store;
   beforeEach(() => {
+    stubRedux();
+    registerReducers({
+      ...reducers
+    });
+
+    store = getStore();
+    store.dispatch(init([], {}, []));
```

Comment:
this may be moot if we are moving off of mount / redux for this test, this line looks safe to remove. not blocking for this PR due to deadline, but worth considering next time you are in here if you are not ripping out redux entirely.

Reply:
I will make a follow up PR to update all the tests.